src/components/background-effect.js

```diff
@@ -20,7 +20,7 @@ export function BackgroundEffect({ type }) {
   }
 
   return (
-    <div className="pointer-events-none fixed h-screen w-screen">
+    <div className="pointer-events-none fixed z-20 h-screen w-screen">
```

Comment:
Right. I've removed it and moved the `<BackgroundEffect />` to the bottom of the videos component. I had to add `inset-0` however, because otherwise the effect wrapper would not begin at the top left of the screen in that case.